c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,69 @@
+<h4 class="text-center text-danger" translate ng-show="feedCtrl.error">Sorry, there was an error while getting the feed.</h4>
+<h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading feed, please wait...</h4>
+<h4 class="text-success text-center" translate ng-show="feedCtrl.noFeed">This user has no recent activity.</h4>
+
+<div ng-show="feedCtrl.documents.length > 0"
+  infinite-scroll="feedCtrl.getDocumentsFromFeed()"
+  infinite-scroll-immediate-check="true"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="feedCtrl.busy || feedCtrl.error || feedCtrl.feedEnd || feedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="feedtable table table-condensed table-bordered table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="feedCtrl.userId" ng-init="user = feedCtrl.userId;" ng-model="amount = feedCtrl.feedLength;">
```

Comment:
Why ``ng-model="amount = feedCtrl.feedLength;"``?
It should be part of the ``ng-init`` I think.